include/nbl/ui/CWindowManagerX11.h

```diff
@@ -21,6 +21,8 @@ NBL_SYSTEM_DECLARE_DYNAMIC_FUNCTION_CALLER_CLASS(X11, system::DefaultFuncPtrLoad
     ,XGetVisualInfo
     ,XCreateColormap
     ,XCreateWindow
+    ,XDestroyWindow
+    ,XCreateSimpleWindow
```

Comment:
whats the difference between `XCreateWindow` and `XCreateSimpleWindow`

Reply:
In `XCreateWindow` you can set more window's attributes.
https://tronche.com/gui/x/xlib/window/XCreateWindow.html

Reply:
ok doesn't seem like non-simple lets us set anything useful.